source/manual/transition-a-site.html.md

```diff
@@ -34,7 +34,22 @@ The content team might ask a developer to offer the department some guidance aro
 
 Follow the instructions in the [transition-config README][transition-config].
 
-### 3) Get a list of old URLs
+### 3) Consider AKA domains
```

Comment:
To save time in the future, can we make this numbered list an actual numbered list (rather than headings) and use the `1.` Markdown trick?

Reply:
Yeah, that's a good shout

Reply:
Needs a rebase now anyway